src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -68101,12 +68114,15 @@ static void M()
                 // (6,34): warning CS1066: The default value specified for parameter 'x' will have no effect because it applies to a member that is used in contexts that do not allow optional arguments
                 //     static partial void G(object x = null, object? y = null) { }
                 Diagnostic(ErrorCode.WRN_DefaultValueForUnconsumedLocation, "x").WithArguments("x").WithLocation(6, 34),
+                // (6,38): warning CS8625: Cannot convert null literal to non-nullable reference type.
+                //     static partial void G(object x = null, object? y = null) { }
+                Diagnostic(ErrorCode.WRN_NullAsNonNullable, "null").WithLocation(6, 38),
```

Comment:
I assume you have a test insertion, since we're adding new warnings?

Reply:
I kicked it off now, thanks. I'll link the insertion once it gets created.

Reply:
https://devdiv.visualstudio.com/DevDiv/_git/VS/pullrequest/286574

Looks like there are 8 new nullable warnings. Some require just adding ? to a CallerMemberName parameter. The rest probably require adding a `!` e.g. `M<T>(T defaultValue = default!)`, because the project targets C# 8 and nullability attributes don't appear to be available.